Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerAnnotator.java

```diff
@@ -91,6 +91,8 @@ public AnnotatorInfo collectInformation(@NotNull final PsiFile psiFile) {
 
     if (!DartAnalysisServerService.getInstance().serverReadyForRequest(module.getProject(), sdk)) return null;
 
+    DartAnalysisServerService.getInstance().virtualFileOpened(annotatedFile);
```

Comment:
I also added a flag not to prioritize files in case of batch inspection run (DartAnalysisServerGlobalInspectionTool), otherwise they never get out of the priority list.